keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -221,106 +218,34 @@ Metadata items:
+
+- `Draft`: The KEP has been proposed and is actively being defined.
+- `Deferred`: The KEP is proposed but not actively being worked on.
+- `Approved`: Using lazy consensus, the approvers have approved this KEP.
+  It will move forward to implementation.
+- `Rejected`: The approvers and authors have decided that this KEP is not moving forward.
+  The KEP is kept around as a historical document.
+- `Withdrawn`: The KEP has been withdrawn by the authors.
+- `Final`: The KEP has been implemented and is no longer actively changed.
+- `Replaced`: The KEP has been replaced by a new KEP.
+  The `superseded-by` metadata value should point to the new KEP.
 
 ### Git and GitHub Implementation
 
-Practically an KEP would be implemented as a pull request to a central repository
-with the following example structure
-
-```
-├── 0000-kep-template.md
-├── CODEOWNERS
-├── index.md
-├── sig-architecture
-│   ├── deferred
-│   ├── orphaned
-│   └── retired
-├── sig-network
-│   ├── deferred
-│   ├── kube-dns
-│   ├── orphaned
-│   └── retired
-├── sig-node
-│   ├── deferred
-│   ├── kubelet
-│   ├── orphaned
-│   └── retired
-├── sig-release
-│   ├── deferred
-│   ├── orphaned
-│   └── retired
-├── sig-storage
-│   ├── deferred
-│   ├── orphaned
-│   └── retired
-├── unsorted-to-be-used-by-newcomers-only
-└── wg-resource-management
-    ├── deferred
-    ├── orphaned
-    └── retired
-```
-
-where each SIG or working group is given a top level directory with subprojects
-maintained by the SIG listed in sub directories. For newcomers to the community
-an `unsorted-to-be-used-by-newcomers-only` directory may be used before an KEP
-can be properly routed to a SIG although hopefully if discussion for a potential
-KEP begins on the mailing lists proper routing information will be provided to
-the KEP author. Additionally a top level index of KEPs may be helpful for people
-looking for a complete list of KEPs. There should be basic CI to ensure that an
-`index.md` remains up to date.
-
-Ideally no work would begin within the repositories of the Kubernetes organization
-before a KEP has been approved by the responsible SIG or working group. While the
-details of how SIGs organize their work is beyond the scope of this proposal one
-possibility would be for each charter SIG to create a top level repository within
-the Kubernetes org where implementation issues managed by that SIG would be filed.
+KEPs are checked into the community repo under the `/kep` directory.
+KEPs can either be put at the top level of that directory or in a SIG specific subdirectory.
+KEPs in SIG specific subdirectories have limited impact outside of the SIG and can leverage SIG specific OWNERS files.
+
+New KEPs can be checked in with a file name in the form of `draft-YYYYMMDD-my-title.md`.
+As significant work is done on the KEP the authors can assign a KEP number.
+This is done by taking the next number in the NEXT_KEP_NUMBER file, incrementing that number, and renaming the KEP.
```

Comment:
seems like something to introduce when (and if) automation assigns the numbers but for O(100) KEPs I think this would be overkill. While some people are able to keep a large mapping of arbitrary numbers to unstructured text in their mind, I believe that in general it would be much more helpful to talk about "cloud provider extraction" than `KEP 0002` in discussions with other humans

Reply:
I disagree here.  Often times we have a lot of similarly named proposals floating around.  Having a completely unambiguous way to talk about a proposal is super useful in my mind.  This is one of the things that is *good* about using github issues as the thing that I'd like to try and keep.

Are you suggesting not having KEP numbers?

Reply:
I'm not suggesting that we should drop KEP numbers. I'm suggesting that in conversations with humans it's probably easier to talk about "implementing the container storage interface for Kubernetes" rather than "implementing KEP 0003". [Rust][] has over 2000 of these type of documents (an order of magnitude more than our design proposals) and seems to be OK. Maybe we should try and get an update from their community on how their process is working for them

[Rust]: https://github.com/rust-lang/rfcs/tree/master/text